src/Mutant.php

```diff
@@ -95,6 +101,36 @@ public function getProcess()
     }
 
     /**
+     * @param $container
+     * @param $group
+     * @param $tracker
+     *
+     * @return MutantResult
+     *
+     * @throws LogicException when the test process is not terminated.
+     */
+    public function getResult($container, $group, $tracker)
```

Comment:
$container is already a member of this class.

Some type hinting would be helpful.